qtop.py

```diff
@@ -1585,7 +1591,7 @@ def get_document(scheduling_system):
                         from xml.etree import ElementTree as etree
 
                 if options.TRANSPOSE:
-                    config['transpose_wn_matrices'] = 'False' if config['transpose_wn_matrices'] == 'True' else 'True'
+                    config['transpose_wn_matrices'] = not config['transpose_wn_matrices']
```

Comment:
@lookfwd remember the conversation we had about this? Now that I have eval'ed it early on, the A = Not A construct is possible. (and don't start yelling about evals, I know, I'll fix them in the future!!)